macros/edr/dbt_artifacts/upload_dbt_models.sql

```diff
@@ -36,8 +36,16 @@
     {% set config_meta_dict = elementary.safe_get_with_default(config_dict, 'meta', {}) %}
     {% set meta_dict = elementary.safe_get_with_default(node_dict, 'meta', {}) %}
     {% do meta_dict.update(config_meta_dict) %}
-    {% set owner = meta_dict.get('owner') %}
-
+    {% set formatted_owner = [] %}
```

Comment:
Owners now will be saved as list (even when given only single owner).
This allows us to handle cases when the users provide owners like:
1. "owner1"
2. "owner1, owner2"
3. ["owner1"]
4. ["owner1", "owner2"]